pkg/nebula/storage_client.go

```diff
@@ -107,40 +94,24 @@ func (s *storageClient) TransLeader(spaceID nebula.GraphSpaceID, partID nebula.P
 		PartID:     partID,
 		NewLeader_: newLeader,
 	}
-	log := getLog().WithValues("req", req)
+	log := getLog()
 	log.Info("start TransLeader")
 	resp, err := s.client.TransLeader(req)
 	if err != nil {
 		log.Error(err, "TransLeader failed")
 		return err
 	}
-	log.Info("TransLeader returned")
 
 	if len(resp.Result_.FailedParts) > 0 {
-		if resp.Result_.FailedParts[0].Code == storage.ErrorCode_E_PART_NOT_FOUND {
+		if resp.Result_.FailedParts[0].Code == nebula.ErrorCode_E_PART_NOT_FOUND {
 			return errors.Errorf("part %d not found", partID)
-		} else if resp.Result_.FailedParts[0].Code == storage.ErrorCode_E_SPACE_NOT_FOUND {
+		} else if resp.Result_.FailedParts[0].Code == nebula.ErrorCode_E_SPACE_NOT_FOUND {
 			return errors.Errorf("space %d not found", partID)
-		} else if resp.Result_.FailedParts[0].Code == storage.ErrorCode_E_TRANSFER_LEADER_FAILED {
+		} else if resp.Result_.FailedParts[0].Code == nebula.ErrorCode_E_LEADER_CHANGED {
 			log.Info("request leader changed",
 				"host", resp.Result_.FailedParts[0].Leader.Host,
 				"port", resp.Result_.FailedParts[0].Leader.Port)
-			leader := fmt.Sprintf("%s:%v", resp.Result_.FailedParts[0].Leader.Host, resp.Result_.FailedParts[0].Leader.Port)
-			if err := s.updateClient(leader); err != nil {
-				return err
-			}
-			resp, err := s.client.TransLeader(req)
-			if err != nil {
-				log.Error(err, "TransLeader failed")
-				return err
-			}
-			log.Info("TransLeader returned")
-			if len(resp.Result_.FailedParts) > 0 {
-				return errors.Errorf("Trans leader failed")
-			}
 			return nil
-		} else if resp.Result_.FailedParts[0].Code == storage.ErrorCode_E_LEADER_CHANGED {
-			return errors.Errorf("leader changed, please request to leader")
```

Comment:
Why remove the logic about `ErrorCode_E_LEADER_CHANGED` ?

Reply:
These codes are unused